Week2/homework/maartjes_work.js

```diff
@@ -44,4 +44,8 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+const earned = tasks.map(task => task.duration / 60)
+  .filter(task => task >= 2)
+  .reduce((sum, task) => sum += task * 16, 0);
```

Comment:
I see you include the hourly salarly(16) in the reduce method. It works and it uses little code, but it's a bit hard to interpret where the 16 is referring to.  I personally would have put it either the 16 in a variable with a clear name or would put in a .map function to make it more clear what is happening, example:

.map(perHourSalary => (perHourSalary * 16))